src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureFilter.cpp

```diff
@@ -202,7 +202,7 @@ namespace OpenMS
               }
             }
 
-            for (const std::pair<String, std::pair<double, double>>& kv : filter_criteria.component_group_qcs.at(cg_qc_it).meta_value_qc)
+            for (const std::pair<const String, std::pair<double, double>>& kv : filter_criteria.component_group_qcs.at(cg_qc_it).meta_value_qc)
```

Comment:
I think `auto` would have been the better choice here
```
for (const auto& kv : ...
```

But there will be a new PR coming up, which fixes a few other things (mostly naming of things as `it` which are not iterators), so just for future reference :)